Makefile

```diff
@@ -46,13 +48,6 @@ tests-etcd:
 	docker rm -f "cilium-etcd-test-container"
 
 tests-consul:
-	@docker rm -f "cilium-consul-test-container" 2> /dev/null || true
```

Comment:
You still need to create the container no? where are the unit tests for consul being tested now?